globals.json

```diff
@@ -143,7 +143,7 @@
   "String.loc": ["@ember/string", "loc"],
   "String.underscore": ["@ember/string", "underscore"],
   "String.w": ["@ember/string", "w"],
-  "Test.Adapter": ["@ember/test/adapter"],
+  "TestAdapter": ["@ember/test/adapter"],
```

Comment:
I think we have `["@ember/test/adapter", null, "TestAdapter"]` for these cases 🤔